optional/fetchmail/fetchmail.py

```diff
@@ -55,7 +55,8 @@ def fetchmail(fetchmailrc):
     with tempfile.NamedTemporaryFile() as handler:
         handler.write(fetchmailrc.encode("utf8"))
         handler.flush()
-        command = FETCHMAIL.format(shlex.quote(handler.name))
+        if "FETCHMAIL_OPTIONS" in os.environ: fetchmail_custom_options = f'{ os.environ["FETCHMAIL_OPTIONS"]}' else fetchmail_custom_options = ""
```

Comment:
```suggestion
        fetchmail_custom_options = os.environ.get("FETCHMAIL_OPTIONS", "")
```
Just shortened this a bit.